md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/WorkflowTypeAPI.java

```diff
@@ -73,6 +71,7 @@ RestWrapper get(
             LOGGER.info("Record with ID:" + workflowTypeId + " selected from WorkflowType by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)